### DISCUSSION TITLE
active app not showing background on app switcher

### DISCUSSION THREAD

**Discussion by walidsaleh** [catppuccin/catppuccin#2518] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

hi, I don't know where to put this ticket.
my machine runs elementary OS 7.1, and when using the theme Catppuccin-Dark, from gnome-look, everything looks great, but the app switcher is not showing the background colour for the active app.

I hope it can be fixed,
many thanks,
excellent job!

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey 👋,

These discussions are reserved for requests to theme an application, not for bugs.

We've archived our port for GTK so you'd be unable to raise an issue in [catppuccin/gtk](https://github.com/catppuccin/gtk). If you're still using the theme from there, we can try to help you in our discord server.

If you're using a Catppuccin gnome theme that's different to the one in catppuccin/gtk, please open a bug report issue from where you downloaded it, thanks!
